src/test/java/dev/openfeature/sdk/ProviderRepositoryTest.java

```diff
@@ -9,6 +9,7 @@
 import static org.awaitility.Awaitility.await;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.atMostOnce;
```

Comment:
nit: I don't think we need this
maybe we can add spotless or checkstyle in our CI

Reply:
nice catch 👍  

We already have checkstyle enabled but the rule to validate the unused imports were missing. I added with this in the PR - https://github.com/open-feature/java-sdk/pull/725/files#diff-b36112ff71ee5b411c9149781cee470c6413dbf498272fa70eb00b8b7555207fR67